fog/view/enclave/api/src/lib.rs

```diff
@@ -162,6 +165,13 @@ pub trait ViewEnclaveApi: ReportableEnclave {
         untrusted_query_response: UntrustedQueryResponse,
     ) -> Result<Vec<u8>>;
 
+    /// Service a query response from a frontend.
+    fn query_backend(
+        &self,
+        payload: EnclaveMessage<NonceSession>,
+        untrusted_query_response: UntrustedQueryResponse,
+    ) -> Result<(Vec<u8>, u64)>;
```

Comment:
:question: perhaps it was due to `fog/view/enclave/src/lib.rs` dealing with the raw values, but was the idea of returning an `EnclaveMessage<>` considered?

Reply:
Yes, and now that I'm thinking about it, James and I discussed modifying this PR to implement that. 

Originally I chose to implement it as a tuple because most Fog enclave methods return bytes rather than the EnclaveMessage. However, it's clearer in this case to return the message. Done.